eras/alonzo/impl/src/Cardano/Ledger/Alonzo/Scripts.hs

```diff
@@ -146,7 +146,7 @@ data Tag
     Mint
   | -- | Validates certificate transactions
     Cert
-  | -- | Validates withdrawl from a reward account
+  | -- | Validates withdrawal from a reward account
```

Comment:
I have a feeling this spelling mistake could be all over the place :grimacing:

Reply:
:rofl: You are right:
```shell
cardano-ledger> git grep withdrawl | wc -l
5
cardano-node> git grep withdrawl | wc -l
2
```

Reply:
Fixed it in all places